app/application.js

```diff
@@ -73,8 +71,10 @@ class RunConditionTableApplication {
     async run() {
         this.logger.info('Starting RCT app...');
         try {
-            await this.databaseManager.sync();
+            await this.databaseManager.migrate();
+            this.databaseService.healthcheckInsertData();
             await this.httpServer.listen();
+            this.buildCli();
```

Comment:
Please refrain from adding unrelated things tho this PR. This is CLI related and it should not go here.